sol/main.c

```diff
@@ -512,6 +512,8 @@ run_sol(struct net_config *net_conf, struct sol_config *sol_conf)
 		goto out;
 	}
 
+	net_conf->gatekeeper_total_pkt_burst += sol_conf->deq_burst_size;
```

Comment:
The line above will be replaced with:
```
net_conf->gatekeeper_total_front_pkt_burst += sol_conf->pri_req_max_len + sol_conf->deq_burst_size;
```
The idea is to account for the packets lingering in the queue of the SOL block. Everything is accounted at the front interface because it's from where the packets come.

Reply:
Shouldn't it be the back interface instead of the front one?

Reply:
The packets are going to the back interface, but they are **allocated** at the front interface.

Given your question, it's worth adding a comment to the code.